command/runner.go

```diff
@@ -203,3 +215,44 @@ func (r *cmdWaitReadCloser) Close() error {
 	r.closed = true
 	return nil
 }
+
+func runPager() (func(), error) {
+	var pager string
+	if p := os.Getenv("KUBEPAGER"); p != "" {
+		pager = p
+	} else if p := os.Getenv("PAGER"); p != "" {
+		pager = p
+	} else {
+		pager = "pager"
```

Comment:
Is "pager" really a good default here?

We could do some `exec.LookPath` to see if the user has `less` or `more` installed, and then use that as a fallback.

But as an absolute fallback, I think you should just return an error of "no pager configured via PAGER or KUBEPAGER environment variables"

Reply:
"pager" seems to be some sort of Unix default. On my Debian it is a symlink to less. I suggest taking the approach of man-db, which seems to check for "pager", "less" and then "more. https://gitlab.com/man-db/man-db/-/blob/main/configure.ac#L121

Reply:
On second thought, this could cause a bad experience for the majority of users, as defaulting to `less` shows the escape codes. So we shouldn't check for "pager" and add the "-R" flag if we default to less.